_tuning-your-cluster/availability-and-recovery/segment-replication/index.md

```diff
@@ -11,17 +11,82 @@ redirect_from:
-{: .warning}
-
 With segment replication, segment files are copied across shards instead of documents being indexed on each shard copy. This improves indexing throughput and lowers resource utilization at the expense of increased network utilization.
 
-As an experimental feature, segment replication will be behind a feature flag and must be enabled on **each node** of a cluster and pass a new setting during index creation.
-{: .note }
-
-### Potential use cases
+## Use cases
 
 - Users who have high write loads but do not have high search requirements and are comfortable with longer refresh times.
 - Users with very high loads who want to add new nodes, as you do not need to index all nodes when adding a new node to the cluster.
 
-This is the first step in a series of features designed to decouple reads and writes in order to lower compute costs.
\ No newline at end of file
+This is the first step in a series of features designed to decouple reads and writes in order to lower compute costs.
+
+## Segment replication configuration
+
+To enable the segment replication type, reference the steps below.
+
+### Enabling the feature flag
+
+There are several methods for enabling segment replication, depending on the install type. You will also need to set the replication strategy to `SEGMENT` when creating the index.
+
+### Enable on a node using a tarball install
+
+The flag is toggled using a new jvm parameter that is set either in `OPENSEARCH_JAVA_OPTS` or in config/jvm.options.
+
+1. Option 1: Update config/jvm.options by adding the following line:
+
+    ````json
+    -Dopensearch.experimental.feature.replication_type.enabled=true
+    ````
+
+1. Option 2: Use the `OPENSEARCH_JAVA_OPTS` environment variable:
+
+    ````json
+    export OPENSEARCH_JAVA_OPTS="-Dopensearch.experimental.feature.replication_type.enabled=true"
+    ````
+1. Option 3: Add the request body parameter when creating an index.
+
+### Enable with Docker containers
+
+If you're running Docker, add the following line to docker-compose.yml underneath the `opensearch-node` and `environment` section:
+
+````json
+OPENSEARCH_JAVA_OPTS="-Dopensearch.experimental.feature.replication_type.enabled=true" # Enables segment replication
+````
+
+### Setting the replication strategy on the index
+
+To set the replication strategy to segment replication, create an index with replication.type set to `SEGMENT`:
+
+````json
+PUT /my-index1
+{
+  "settings": {
+    "index": {
+      "replication.type": "SEGMENT" 
+    }
+  }
+}
+````
+
+## Further resources regarding segment replication
+
+1. [Known issues](https://github.com/opensearch-project/OpenSearch/issues/2194).
```

Comment:
This is not valid any more. may be you can replace it with:
To track future enhacements on the Segment replication post GA: https://github.com/orgs/opensearch-project/projects/99